src/test/java/com/transferwise/openbanking/client/api/event/RestEventClientTest.java

```diff
@@ -0,0 +1,215 @@
+package com.transferwise.openbanking.client.api.event;
+
+import static com.transferwise.openbanking.client.test.factory.AccessTokenResponseFactory.aAccessTokenResponse;
+import static com.transferwise.openbanking.client.test.factory.SoftwareStatementDetailsFactory.aSoftwareStatementDetails;
+
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscription1;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscription1Data;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionResponse1;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionResponse1Data;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionsResponse1;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionsResponse1Data;
+import com.transferwise.openbanking.client.api.payment.v3.model.event.OBEventSubscriptionsResponse1DataEventSubscription;
+import com.transferwise.openbanking.client.configuration.AspspDetails;
+import com.transferwise.openbanking.client.configuration.SoftwareStatementDetails;
+import com.transferwise.openbanking.client.json.JacksonJsonConverter;
+import com.transferwise.openbanking.client.json.JsonConverter;
+import com.transferwise.openbanking.client.jwt.JwtClaimsSigner;
+import com.transferwise.openbanking.client.oauth.OAuthClient;
+import com.transferwise.openbanking.client.oauth.domain.AccessTokenResponse;
+import com.transferwise.openbanking.client.test.factory.AspspDetailsFactory;
+import java.util.List;
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.client.MockRestServiceServer;
+import org.springframework.test.web.client.match.MockRestRequestMatchers;
+import org.springframework.test.web.client.response.MockRestResponseCreators;
+import org.springframework.web.client.RestTemplate;
+
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings({
+    "PMD.AvoidDuplicateLiterals",
+    "checkstyle:membername",
+    "checkstyle:variabledeclarationusagedistance",
+    "checkstyle:methodname",
+    "checkstyle:abbreviationaswordinname"})
```

Comment:
As you are adding this test can you fix "checkstyle:variabledeclarationusagedistance" to not mute it?

Reply:
Removed in [7ac1674](https://github.com/transferwise/openbanking-client/pull/170/commits/7ac1674d34c7a2e6229e97a60602fe25f6e2302a).